OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -52,147 +52,601 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    </layout>
+                                </layoutDesc>
+                            </objectDesc>
+                            
+                            <handDesc>
+                                <handNote xml:id="h1" script="Ethiopic">
+                                    <locus from="1ra" to="123vb"/>
+                                    <seg type="script"/>
+                                    <seg type="ink">Black, red or deep vermilion</seg>
+                                    <date notBefore="1500" notAfter="1600"></date>
+                                    <desc>
+                                        Fine and trained hand. Characters are regularly spaced.
+                                    </desc>
+                                    <note>
+                                        The script displays some relatively archaic features: the numeral <foreign xml:lang="gez">፮</foreign> has a compressed shape and the open loop; 
+                                        the letter መ has the loops not completely separated; the word <foreign xml:lang="gez">እግዚአ፡ ብሔር፡</foreign> is sometimes written as two separate words. 
+                                        The graphemes ግ and ዚ are occasionally written with ligature in the word <foreign xml:lang="gez">እግዚእነ፡</foreign>, e.g. on <locus target="#35va #100vb"/>. 
+                                        One bottom ruled line has been left unwritten on leaves <locus from="24va" to="25vb"/>.
+                                    </note>                              
+                                    <seg type="rubrication">
+                                        Holy names; several groups of lines on the incipit page of each Gospel, alternating with black lines;                                    
+                                        the numbers of the chapters in the list of the <foreign xml:lang="la">tituli</foreign> at the beginning of each Gospel;
+                                        the numbers of the Eusebian Canons to which the Ammonian sections, penned in black ink in the left margins immediately above those numbers, belong;
+                                        the numbers and titles of the chapters are written in the upper or lower margin, in correspondence of the beginning of the chapters;
+                                        the stichometric note at the end of the Gospels of Matthew, on <locus target="#35ra"/>;
+                                        the captions <foreign xml:lang="gez">ማርቆስ፡</foreign> on <locus target="#35vb"/>, <foreign xml:lang="gez">ስዕለ፡ ሉቃስ፡</foreign> (partly illegible) 
+                                        on <locus target="#61v"/>, and <foreign xml:lang="gez">ዮሐንስ፡</foreign> on <locus target="#100va"/>;
+                                        elements of the numerals and of the punctuation signs, including text dividers and caesura signs
```

Comment:
"caesura" -> This might require changing based on the conversation above.

Reply:
@SusanneHummel matter for next meeting: sequence of the rubricated items.